pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -83,6 +83,13 @@ var _ = Describe("Qemu agent poller", func() {
 			agentStore.Store(GET_OSINFO, fakeInfo)
 			Expect(agentStore.AgentUpdated).ToNot(Receive())
 		})
+
+		It("should load nil slice when no interfaces exists", func() {
+			var agentStore = NewAsyncAgentStore()
+			agent := agentStore.GetInterfaceStatus()
+
+			Expect(agent == nil).To(Equal(true))
```

Comment:
I do not thing `agent` fits here, how about `interfacesStatus`?

And the assert: `Expect(interfacesStatus).To(BeNil())`